src/abstractdataframe/io.jl

```diff
@@ -232,8 +232,8 @@ function Base.show(io::IO, ::MIME"text/latex", df::AbstractDataFrame)
     write(io, "\t& ")
     header = join(map(c -> latex_escape(string(c)), cnames), " & ")
     write(io, header)
-    write(io, "\\\\ \n")
```

Comment:
I think it was just to make it a bit more readable than having 5 backslashes in a row.